### PR TITLE
zrepl 0.2.0

### PR DESCRIPTION
**Pull request by JMoVS**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by JMoVS** [comment]
successor to https://github.com/Homebrew/homebrew-core/pull/45233 without the gitbhub-bug commit

File: Formula/zrepl.rb [hunk 1]
```diff
@@ -28,13 +28,13 @@ def install
 
     ENV["GOPATH"] = gopath
     ENV.prepend_create_path "PATH", gopath/"bin"
+    ENV["GO111MODULE"] = "on"
```
Comment (chenrui333) [inline Formula/zrepl.rb]: we recently removed this option in lots of golang formula. So I think this line can be removed.

Comment (chenrui333) [reply Formula/zrepl.rb]: Relates to https://github.com/Homebrew/homebrew-core/pull/44565

Comment (JMoVS) [reply Formula/zrepl.rb]: I've removed it, thx

**Comment by SMillerDev** [comment]
@brewtestbot test this please!

**Review by chenrui333 — APPROVED** [review]
Thanks a lot!

**Pull request closed without merging**